plans/main.fmf

```diff
@@ -3,21 +3,18 @@ discover:
 execute:
     how: tmt
 prepare:
-    how: feature
-    epel: enabled
-
-/upstream:
-    summary: Run SELinux specific Podman tests on upstream PRs
-    discover+:
-        filter: tag:upstream
-    adjust+:
-        enabled: false
-        when: initiator is not defined or initiator != packit
-
-/downstream:
-    summary: Run SELinux specific Podman tests on bodhi / errata and dist-git PRs
-    discover+:
-        filter: tag:downstream
-    adjust+:
-        enabled: false
-        when: initiator == packit
+    - when: distro == centos-stream or distro == rhel
```

Comment:
The removal of `/upstream` and `/downstream` is intentional as we can run the same set of tests across the board.